blog/2025-03-26-pixi-react-v8-launches.md

```diff
@@ -0,0 +1,89 @@
+
+<!--truncate-->
+
+## A Fresh Start
+
+The decision to completely rebuild @pixi/react was driven by several key factors:
+
+* Pixi.js v8 itself underwent a significant rewrite, which would have required major rework in the Pixi React v7 codebase regardless
+* Updating to the latest version of react-reconciler needed for React 19 support would have broken most of the existing implementation
+* A fresh start gave us the opportunity to reimagine the library's ergonomics, allowing the implementation of a new pragma and eliminating the wrapper components of previous versions in favor of directly exposing Pixi.js components through JSX proxies
+
+This complete rewrite is heavily inspired by [`@react-three/fiber`](https://github.com/pmndrs/react-three-fiber/tree/master), and would not have been possible without the significant contributions from the R3F maintainers and community.
+
+## What's New?
+
+### Custom Pragma
+
+At the heart of Pixi React v8 lies a new JSX pragma that greatly simplifies the ergonomics of using the library. Now, all Pixi.js components are directly reflected as JSX components with the `pixi` prefix:
+
+```jsx
+import { Application } from '@pixi/react'
+import { Sprite } from 'pixi.js'
+
+function MyComponent() {
+  const draw = useCallback(graphics => {
+    graphics.setFillStyle({ color: 'red' })
+    graphics.rect(0, 0, 25, 25)
+  }, [])
+
+  const texture = useMemo(() => Assets.get('bunny.png'), [])
+
+  return (
+    <Application>
+      <pixiSprite x={100} y={100} texture={texture} />
+      <pixiGraphics x={0} y={0} draw={graphics => {
+        graphics.setFillStyle({ color: 'red' })
+        graphics.rect(0, 0, 25, 25)
+      }} />
+    </Application>
+  )
+}
+```
+
+This approach eliminates the need for custom component wrappers, making the library much more future-proof and easier to maintain. As Pixi.js evolves, @pixi/react will automatically support new components without requiring updates to the library itself.
+
+### Full TypeScript Support
+
+Pixi React v8 was rebuilt from the ground up with TypeScript, providing autocompletion, type checking, and an improved developer experience even for those that aren't using Typescript.
+
+### Pixi.js v8 Compatible
+
+Built specifically for Pixi.js v8, this release brings all the performance improvements, WebGPU support, and modern JavaScript features from the core library directly to your React applications.
+
+### React 19 Support
+
+React 19 represents a massive shift in the way React works under-the-hood, which means that libraries that are deeply integrated with React have to follow suit. While we recognize that this places significant hurdles in the upgrade path for our React 18 users, we believe that being able to take advantage of the latest React features and optimizations is worth the effort.
+
+## Getting Started
+
+Ready to dive in? Check out the [new docs site](https://react.pixijs.io/) for Pixi React v8!
```

Comment:
I think it would be good to call out the CLI tool which has a template for getting you up and running quickly

```suggestion
Ready to dive in? Check out the [new docs site](https://react.pixijs.io/) for Pixi React v8!

Want to get started even faster? Use the [PixiJS Create](https://pixijs.io/create-pixi/i) CLI spin up a Pixi React project in seconds:

`npm create pixi.js@latest --template framework-react`

This sets up a ready-made project with all the essentials, so you can jump straight into building.
```